hawkbit-repository/hawkbit-repository-jpa/src/test/java/org/eclipse/hawkbit/repository/jpa/SystemManagementTest.java

```diff
@@ -109,26 +100,27 @@ public void systemUsageReportCollectsActionsOfAllTenants() throws Exception {
 
         for (int i = 0; i < tenants; i++) {
             final String tenantname = "tenant" + i;
-            securityRule.runAs(WithSpringAuthorityRule.withUserAndTenant("bumlux", tenantname), () -> {
-                systemManagement.getTenantMetadata(tenantname);
-                if (artifactSize > 0) {
-                    createTestArtifact(random);
-                    createDeletedTestArtifact(random);
-                }
-                if (targets > 0) {
-                    final List<Target> createdTargets = createTestTargets(targets);
-                    if (updates > 0) {
-                        for (int x = 0; x < updates; x++) {
-                            final DistributionSet ds = testdataFactory.createDistributionSet("to be deployed" + x,
-                                    true);
-
-                            deploymentManagement.assignDistributionSet(ds, createdTargets);
+            securityRule.runAs(WithSpringAuthorityRule.withUserAndTenant("bumlux", tenantname, true, true,
+                    SpringEvalExpressions.SYSTEM_ROLE), () -> {
+                        systemManagement.getTenantMetadata(tenantname);
```

Comment:
Same here would prefer more methods, because it's hard to understand, but I think it's not your code ;).